### PR TITLE
Update HeadingSection.js

### PR DESCRIPTION
**Pull request by shretimag**
Formatted by prettier

### CODE REVIEW

File: src/components/InsightsComponents/HeadingSection.js
Comment (Mayank2602) [inline src/components/InsightsComponents/HeadingSection.js:1]: Can we import the styled components that are being used everywhere from a single file. Eg. Container, heading etc.
Check https://stackoverflow.com/questions/52991728/how-to-export-styled-component-from-a-file.

**Review by Mayank2602 — CHANGES_REQUESTED** [review]

File: src/components/InsightsComponents/HeadingSection.js
Comment (shretimag) [reply src/components/InsightsComponents/HeadingSection.js:1]: okay sir will try

Comment (shretimag) [reply src/components/InsightsComponents/HeadingSection.js:1]: @Mayank2602 sir can I also change the files insightinternship.js and insightinternship2022.js and import the components repeated from a single file as you said, cause I don't want to create confusion among the person who is assigned for that file.

Comment (shretimag) [reply src/components/InsightsComponents/HeadingSection.js:1]: Sir i have made changes in my assigned files, if you approve i can put other component styles and change few files more which have repeated styled components

**Pull request merged**